torchmultimodal/modules/encoders/albef_vision_encoder.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from functools import partial
+from typing import Callable
+
+import torch
+from torch import nn
+from torchvision.models.vision_transformer import VisionTransformer
+
+
+class ALBEFVisionEncoder(VisionTransformer):
+    def __init__(
```

Comment:
Make sure to add the docstring when you get a chance

Reply:
Agree, having a docstring will be good